packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -580,8 +580,6 @@ export class RunningSummarizer implements IDisposable {
             return;
         }
 
-        this.summarizeTimer.clear();
```

Comment:
It was intentionally cleared after the runtime finished generating the summary. It isn't meant to fire while waiting for summarize op broadcast or summary ack/nack, since we have pending summary timeout for that purpose.

Reply:
I think what you are saying is that summarize() should not set/reset timer, instead it should be done in this function. Right?

Reply:
@arinwt , please let me know what you think

Reply:
Yeah I think it's just in the outer function to make sure it gets canceled in any unknown error case. Feel free to move it lower if you want. It was just meant to capture time from start of the call until client-side work is done (generate summary tree primarily).

Reply:
Looking a little closer, yeah I think we do very little between `summarize()` and `generateSummary` so that seems totally reasonable to move it to `generateSummaryWithLogging`.